taxi_service/settings.py

```diff
@@ -43,8 +43,8 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "debug_toolbar",
-    "crispy_forms",
     "taxi",
+
```

Comment:
The 'INSTALLED_APPS' list includes 'debug_toolbar', which is typically used only in development environments. Make sure to remove or conditionally include it for production settings.